plugins-compat-tester/src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -49,6 +51,12 @@ public void run(Config config, File baseDirectory, File buildLogFile, String...
             cmd.add("--define=" + entry);
         }
         cmd.addAll(Arrays.asList(goals));
+        for (String goal: goals){
```

Comment:
Can be just hardcoded above for all invocations. E.g. like ` cmd.add("--batch-mode");` above